parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -1,13 +1,8 @@
 echo $(date),$(hostname) > /var/log/azure/cluster-provision-cse-output.log;
-for i in $(seq 1 1200); do
-grep -Fq "EOF" /opt/azure/containers/provision.sh && break;
-if [ $i -eq 1200 ]; then exit 100; else sleep 1; fi;
-done;
+cloud-init status --wait  > /dev/null 2>&1;
+[ $? -ne 0 ] && echo 'cloud-init failed' >> /var/log/azure/cluster-provision-cse-output.log && exit 1;
+echo "cloud-init succeeded" >> /var/log/azure/cluster-provision-cse-output.log;
 {{if IsAKSCustomCloud}}
-for i in $(seq 1 1200); do
-grep -Fq "EOF" {{GetInitAKSCustomCloudFilepath}} && break;
```

Comment:
can we confirm when this was moved to VHD? and potentially base64 encode/write during CSE if needed?

Reply:
oh nvm. you're only removing the wait here, not removing cloud init

Reply:
yes, just removing the wait, not from cloud init.